pkg/gtest/gocovertest.go

```diff
@@ -129,7 +129,18 @@ func checkTestFileExistence(folder string) (bool, error) {
 
 var (
 	ErrPackageNameNotFound = errors.New("package not found")
-	packageRegexp          = regexp.MustCompile(`^package\s+([a-zA-Z][a-zA-Z0-9]*)`) // regexp for matching "package xxx"
+
+	// The rule of package name:
+	// https://go.dev/ref/spec#Package_clause
+	//
+	//   PackageClause  = "package" PackageName .
+	//   PackageName    = identifier .
+	//
+	//   identifier     = letter { letter | unicode_digit } .
+	//   letter         = unicode_letter | "_" .
+	//   unicode_letter = /* a Unicode code point categorized as "Letter" */ .
+	//   unicode_digit  = /* a Unicode code point categorized as "Number, decimal digit" */ .
+	packageRegexp = regexp.MustCompile(`^package\s+([a-zA-Z_][a-zA-Z0-9_]*)`) // regexp for matching "package xxx"
```

Comment:
add some negative case test?

Reply:
let's release a new version to mitigate the issue first, and find a better solution, say like the mockgen tool to get the GOPACKAGE name,